ddl/ddl_api.go

```diff
@@ -1447,9 +1447,14 @@ func buildTableInfo(
 					}
 				}
 			case model.PrimaryKeyTypeDefault:
+				// AlterPrimaryKey = true ----> all pk must be nonclustered
+				// AlterPrimaryKey = false + EnableClusteredIndex = true + noBinlog ---> all pk must be clustered
```

Comment:
(AlterPrimaryKey = false) + (EnableClusteredIndex = true) + noBinlog ---> all pk must be clustered